Failed to build with GCC 11

svt-vp9 fail to build with GCC 11 at Fedora.

```
/usr/bin/gcc  -ISource/Lib/ASM_SSSE3 -I../Source/Lib/ASM_SSSE3 -I../Source/API -I../Source/Lib/Codec -I../Source/Lib/VPX -I../Source/Lib/C_DEFAULT -I../Source/Lib/ASM_SSE2 -I../Source/Lib/ASM_SSE4_1 -I../Source/Lib/ASM_AVX2 -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64  -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -Wall -Wextra -Wformat -Wformat-security -fPIE -fPIC -D_FORTIFY_SOURCE=2 -fvisibility=hidden /GS -DNDEBUG -fPIC -std=gnu99 -MD -MT Source/Lib/ASM_SSSE3/CMakeFiles/VP9_ASM_SSSE3.dir/inv_txfm_ssse3.c.o -MF Source/Lib/ASM_SSSE3/CMakeFiles/VP9_ASM_SSSE3.dir/inv_txfm_ssse3.c.o.d -o Source/Lib/ASM_SSSE3/CMakeFiles/VP9_ASM_SSSE3.dir/inv_txfm_ssse3.c.o -c ../Source/Lib/ASM_SSSE3/inv_txfm_ssse3.c
In file included from ../Source/Lib/ASM_SSSE3/inv_txfm_ssse3.c:11:
../Source/Lib/ASM_SSSE3/inv_txfm_ssse3.h: In function 'idct8x8_12_add_kernel_ssse3':
/usr/lib/gcc/x86_64-redhat-linux/11/include/tmmintrin.h:124:1: error: inlining failed in call to 'always_inline' '_mm_mulhrs_epi16': target specific option mismatch
  124 | _mm_mulhrs_epi16 (__m128i __X, __m128i __Y)
      | ^~~~~~~~~~~~~~~~
```
Full build log https://kojipkgs.fedoraproject.org//work/tasks/2850/60932850/build.log

Please help me solve this problem.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -119,11 +119,9 @@ set(flags_to_test
     -fPIC
     -D_FORTIFY_SOURCE=2
     #-flto
-    -fvisibility=hidden
-    /GS
-    /sdl)
+    -fvisibility=hidden)
 if(MSVC)
-    list(APPEND flags_to_test /MP)
+    list(APPEND flags_to_test /MP /GS /sdl)
 else()
     option(NATIVE "Build for native performance (march=native)")
     list(INSERT flags_to_test 0 -Wall)

